go.mod

```diff
@@ -4,6 +4,9 @@ go 1.14
 
 require (
 	github.com/go-jose/go-jose/v3 v3.0.0
-	golang.org/x/net v0.0.0-20220826154423-83b083e8dc8b
-	golang.org/x/oauth2 v0.0.0-20220822191816-0ebed06d0094
+	github.com/kr/pretty v0.1.0 // indirect
+	github.com/stretchr/testify v1.7.0 // indirect
+	golang.org/x/net v0.4.0
+	golang.org/x/oauth2 v0.3.0
```

Comment:
big +1 on this change, it drops a ton of transitive dependencies out of the graph